MobileFilter/sections/specific_web.txt

```diff
@@ -1108,6 +1108,7 @@ pikabu.ru##.ads-under-post
 dailyportalz.jp##main > div > div.contents_title:first-child
 ||cpcdn.com/assets/device/ad/$domain=cookpad.com
 cookpad.com##.standard_ad_slot
+cookpad.com##div > div[data-ad-slot^="spweb-"]
```

Comment:
As I can see, there is no difference whether to use `div >` or not , so you can remove this part.
```
cookpad.com##div[data-ad-slot^="spweb-"]
```

Reply:
OK, I will fix.